src/lib/routes.ts

```diff
@@ -1,6 +1,14 @@
 export const SIGN_IN = '/sign-in'
-export const SIGN_IN_SELECT_PROFILE_SUBROUTE = '/select-profile'
+export const SIGN_IN_SELECT_PROFILE = '/sign-in/select-profile'
 
 export const HOME = '/home'
 export const PROFILE = '/profile'
 export const SETTINGS_PROFILE = '/settings/profile'
+
+export const AllRoutes = {
```

Comment:
nit: since this is a constant, should be in UPPER_CAMEL_CASE

```suggestion
export const ALL_ROUTES = {
```